tslint.json

```diff
@@ -1,5 +1,5 @@
 {
     "extends": ["tslint:latest", "tslint-config-prettier"],
     "rulesDirectory": ["tslint-plugin-prettier"],
-    "rules": {"prettier": true}
-}
\ No newline at end of file
+    "rules": {"prettier": true, "no-submodule-imports": false}
```

Comment:
It looks like [this config value](https://palantir.github.io/tslint/rules/no-submodule-imports/) accepts an allow list of submodules, so maybe we can have our cake and eat it too?

Something like:
```
"rules": {"prettier": true, "no-submodule-imports": [true, "fs/promises"]}
```

Thoughts?